src/rpc.ts

```diff
@@ -82,6 +82,8 @@ export class RPC extends EventEmitter {
   private remoteCallQueue: RPCMessageWithCounter<any>[] = [];
   private lastSequentialCall = -1;
   private remoteProtocolVersion: string | undefined;
+  private handler: Window | Document = window;
+  private dataIsObject: boolean = false;
```

Comment:
Does this need to be a member variable?

Reply:
dataIsObject just keeps reference on whether or not we should JSON.parse or not